### PR TITLE
Pull request semana 2

### PR DESCRIPTION
**Pull request by diirf**
Actualizo las etiquetas semánticas en los ficheros HTML, renombro el archivo CSS con el nombre requerido y actualizo su contenido con las nuevas etiquetas.

### CODE REVIEW

File: student-cvs/diirfv-7677/focused.png
Comment (Josofico) [inline student-cvs/diirfv-7677/focused.png:1]: Will be a good practice that you make a dir called "Assets" where you can insert your pictures.jpg or .png.

File: student-cvs/diirfv-7677/main.js
Comment (Josofico) [inline student-cvs/diirfv-7677/main.js:1]: I think that this file doens't belong here. It's a file for week 3

File: student-cvs/diirfv-7677/cv.html [hunk 1]
```diff
@@ -4,17 +4,19 @@
...
     <title>Indira Franchi</title>
 </head>
 
 <body>
     <header>
         <nav class="header-nav">
             <ul>
-                <li><a href="../index.html">INICIO</a></li>
-                <li><a href="./projects.html">PROYECTOS</a></li>
+                <li><a href="/">Inicio</a></li>
```
Comment (Josofico) [inline student-cvs/diirfv-7677/cv.html:17]: Capitalize > Uppercase.

File: student-cvs/diirfv-7677/cv.html [hunk 2]
```diff
@@ -49,7 +51,7 @@ <h1>Formación complementaria</h1>
         <section>
             <h1>Idiomas</h1>
             <ul>
-                <li>Italiano <strong>A2 en curso</strong></li>
+                <li>Italiano <strong>A2</strong><em> en curso</em></li>
```
Comment (Josofico) [inline student-cvs/diirfv-7677/cv.html:54]: I'd like to see your websites in spanish, english and italian :O !!

**Review by Josofico — COMMENTED** [review]
Hi! You can see that I'm practice my english so I decided to make a review in this language.

I drop some advices about good practice, so you can wirte code better and easier to read.

Do you know how can I see your website? I mean, your index literally like you did it.

File: student-cvs/diirfv-7677/focused.png
Comment (diirf) [reply student-cvs/diirfv-7677/focused.png:1]: You are right!, now you can see the folders

File: student-cvs/diirfv-7677/cv.html (hunk 1 above)
Comment (diirf) [reply student-cvs/diirfv-7677/cv.html]: Yes, in my first commit I wrote with capitalize but after a while, I decided to use uppercase

File: student-cvs/diirfv-7677/cv.html (hunk 2 above)
Comment (diirf) [reply student-cvs/diirfv-7677/cv.html]: Ha ha ha, ok, in the future I will do it when we learn how to change the language depends of the origin of the user

**Comment by diirf** [comment]
Thank you for the feedback. Regarding your question about how to see my website, you can go to my GitHub profile and dowload my folder of the week two (the green button with "clone" on it)

**Review by Josofico — APPROVED** [review]

**Comment by sheifg** [comment]
Hola Indira,
me ha gustado mucho como lo has hecho y también lo de agregar las carpetas para tenerlo mejor organizado, como te ha recomendado @Josofico :)
PD: Si tenéis tiempo y si os apetece, os agradecería que me revisarais mi PR de la semana 2.

**Review by sheifg — APPROVED** [review]

**Pull request merged**